openwisp_notifications/handlers.py

```diff
@@ -316,3 +317,19 @@ def schedule_object_notification_deletion(instance, created, **kwargs):
         tasks.delete_ignore_object_notification.apply_async(
             (instance.pk,), eta=instance.valid_till
         )
+
+
+def register_notification_observation(model, signal):
+    signal.connect(
+        update_notification_cache,
+        sender=model,
+        dispatch_uid='{}_{}_update_notification_cache'.format(
+            str(signal), model.__name__
```

Comment:
Can't we use `f-strings` here?

Reply:
I agree.

Reply:
another thing, are we sure `str(signal)` returns the name of the signal? Eg: `post_save`. Please double check this.

Reply:
> Can't we use f-strings here?

Yes we can use it. But in most of the places `.format()` is being used. 

> another thing, are we sure str(signal) returns the name of the signal? Eg: post_save. Please double check this.

No, It do not return signal name. I have added another parameter named `signal_uid` for this.

Reply:
> Yes we can use it. But in most of the places `.format()` is being used.
>

Then we should open an issue for this. :thinking:

Reply:
I don't think so. It is present in many modules. I have seen it. Then we will need to open an issue in each module. I thinks it's not harming code in anyway. To do this we must have a strong reason for it.